packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -216,15 +216,15 @@ class IosProject extends XcodeBasedProject {
   }
 
   /// The bundle name of the host app, `My App.app`.
-  Future<String?> hostAppBundleName(BuildInfo buildInfo) async {
+  Future<String?> hostAppBundleName(BuildInfo? buildInfo) async {
```

Comment:
Made this nullable to handle the null `BuildInfo` piped down from the install command.